middlewares/auth.js

```diff
@@ -0,0 +1,17 @@
+const modelUser = require('../models/user')
+
+module.exports.auth = async (req, res, next) => {
+  try {
+    const tokenUser = req.get('Authorization')
+    const User = await modelUser.findOne({ token: tokenUser })
```

Comment:
I think we should not use capital letter with local variables, we must const
`const user = await ModelUser.findOne({ token: tokenUser })`

Reply:
Ok, issue solved!